include/swift/AST/Module.h

```diff
@@ -744,7 +748,7 @@ class FileUnit : public DeclContext {
   // Only allow allocation of FileUnits using the allocator in ASTContext
   // or by doing a placement new.
   void *operator new(size_t Bytes, ASTContext &C,
-                     unsigned Alignment = alignof(FileUnit));
+                     unsigned Alignment = alignof(FileUnit&));
```

Comment:
Definitely not correct, except coincidentally. Also confused; this operator occurs within the definition of FileUnit, so I would expect it to be able to calculate the alignment.

…well, okay, maybe not, that might require two passes. \*goes to check C++ standard\*